src/rlx_config.erl

```diff
@@ -256,10 +256,14 @@ load_terms({vm_args, false}, {ok, State}) ->
     {ok, rlx_state:vm_args(State, false)};
 load_terms({vm_args, VmArgs}, {ok, State}) ->
     {ok, rlx_state:vm_args(State, filename:absname(VmArgs))};
+load_terms({vm_args_src, VmArgs}, {ok, State}) ->
+    {ok, rlx_state:vm_args_src(State, filename:absname(VmArgs))};
```

Comment:
is this added for symmetry? I was under the impression that the file already behaved that way?

Reply:
It already partially behaves that way. The difference is no need to set RELX_REPLACE_OS_VARS and it doesn't have to be copied to `vm.args.org` so works a little more smoothly with symlinks.

Reply:
Got it; avoiding RELX_REPLACE_OS_VARS is a nice addition.